lib/utils/session_vars.js

```diff
@@ -24,6 +24,11 @@ module.exports = session_vars;
 // These default to null
 // TODO: How to isolate debug elsewhere so this file can use the logger?
 session_vars.get_debug = function () { return process.env.DEBUG || null; };
+session_vars.get_dangerous_debug = function () {
+  /** Note this will cause failure of tests that reassure the user about
+  *   security precautions we took, like not taking a screenshot. */
+  return process.env._ALKILN_DANGEROUS_DEBUG || null;
+};
```

Comment:
Remove dangerous debug, but make commits that add and then remove this so that it can be referenced in an issue describing how to do this.

Discussion:
> Added _ALKILN_DANGEROUS_DEBUG for taking pictures of passwords and getting values of secret variables as it became very difficult to debug our tests without that.
> 
> Response: somewhat defeats the purpose of secret stuff, or at least makes them less safe.
> 
> Conclusion: remove code, but make an issue/document the link to the commits. Maybe we add it and remove it. May not notice dangerous screenshots for a few weeks.
> 
> Suggestion: Can we just change the session vars function to always return `null` so we don't have to remove the infrastructure.
> 
> Response: The infrastructure is the concern. An `if` could get mistyped in a future change.
> 
> Question: Can we provide a link to the code we're going to remove so it's easier to redo.
> 
> Response: yes, or make a branch that always has this, but the commit might be better.

Reply:
Issue for reference: https://github.com/SuffolkLITLab/ALKiln/issues/813